.github/workflows/test_ci.yml

```diff
@@ -0,0 +1,30 @@
+name: Go Test
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        go: [1.21,"1.20"]
+
+    steps:
+    - name: Set up Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: ${{ matrix.go }}
+      id: go
+
+    - name: Check out code
+      uses: actions/checkout@v4
+
+    - name: Install dependencies
+      run: |
+        sed -i 's/1.21/${{matrix.go}}/g' go.mod
```

Comment:
Con estas cosas me ponéis en un dilema, Alejandro. De veras que preferiría que buscárais otra solución al mismo problema. Una simple es tener varias versiones del fichero go.mod que se llamen go-1.21.mod y go-1.20.mod, por ejemplo, y copiarlas dependiendo del valor de ${{matrix.go}}.  Otra, crear el fichero sobre la marcha usando una plantilla o simplemente `echo`. En fin, hay diferentes soluciones posibles y el que busques una solución da indicación de que te has preocupado de entender cuál es el problema, qué significa `${{matrix.go}}`, y otras posibles soluciones usando la sintaxis del workflow.
En todo caso, dependiendo de cuál sea la versión que tienes en el contenedor, puede que uno de ellos no haga falta.

Reply:
Esto al final, al ejecutar los tests otra vez me he dado cuenta que habia un error, y he aprovechado para cambiarlo. Arreglado en [0689038](https://github.com/adelahera/basket-stats/pull/45/commits/0689038533b0b3c5accd488555b4c51c8932f734)